gax-httpjson/src/test/java/com/google/api/gax/httpjson/ApiMessageHttpRequestTest.java

```diff
@@ -143,6 +143,9 @@ public String getFieldValue(String s) {
     HttpRequest httpRequest = httpRequestRunnable.createHttpRequest();
     String expectedUrl = ENDPOINT + "name/tree_frog" + "?requestId=request57";
     Truth.assertThat(httpRequest.getUrl().toString()).isEqualTo(expectedUrl);
+    Truth.assertThat(httpRequest.getHeaders().getContentLength())
+        .isEqualTo(httpRequest.getContent().getLength());
+    Truth.assertThat(httpRequest.getHeaders().getContentLength()).isGreaterThan((long) 0);
```

Comment:
Please use `0L` literal instead of `(long) 0`

Reply:
Removed this .